translator/tocwconfig/sampleConfig/appsignals_and_eks_config.yaml

```diff
@@ -28,6 +28,12 @@ exporters:
                     - aws.span.kind
                   regex: ^(SERVER|LOCAL_ROOT)$
                   separator: ;
+                - label_names:
+                    - EC2.InstanceId
+                    - EC2.AutoScalingGroupName
+                    - host.name
+                  regex: .*
```

Comment:
I trying to recall why do we need to add these labels here? it is not being used as metric dimensions which should be auto-populated as EMF regular attributes in my mind. 
In this change, we have `regex: .*`, what if otel raw metric data doesn't have   `EC2.AutoScalingGroupName` existed.

Reply:
I am pretty sure now that we should not have these rules under `label_mathers`, these rules are used to for extracting certain metrics from the matched label values. We don't actually need it for our cases. :)

```
- label_names:
    - EC2.InstanceId
    - EC2.AutoScalingGroupName
    - host.name
   regex: .*
```

Reply:
got it, yeah I was thinking about this too. We also have k8s attributes that didn't have to be added to the emf config. Probably just the processor change was enough. Will remove